pages/deposits_and_withdrawals/cli_commands.md

```diff
@@ -0,0 +1,27 @@
+## CLI commands
+
+### Deposit to Subaccount
```

Comment:
here and below: for each section, can you add a short description of each what the actions are?

Reply:
for example, this allows depositing into subaccount